.github/workflows/test.yml

```diff
@@ -0,0 +1,12 @@
+name: Test
+on: [push]
```

Comment:
```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

that should fix it.

Reply:
That it does not ran inside this Pull Request. They actually allow it, a few days ago it was changed, that the maintainer needs to approve it but generally it's possible.

Yes, mostly such a proposed way is used to also run it when people create pull requests.

Reply:
Sweet, thanks!